lib/cartopy/mpl/geocollection.py

```diff
@@ -36,3 +36,11 @@ def set_array(self, A):
         # Now that we have prepared the collection data, call on
         # through to the underlying implementation.
         super(QuadMesh, self).set_array(A)
+
+    def set_clim(self, vmin=None, vmax=None):
+        # Update _wrapped_collection_fix color limits if it is there.
+        if hasattr(self, '_wrapped_collection_fix'):
+            self._wrapped_collection_fix.set_clim(vmin, vmax)
+
+        # Update color limits for the rest of the cells.
+        super(QuadMesh, self).set_clim(vmin, vmax)
```

Comment:
Since we're only on Python 3 now this can just be:
```python
super().set_clim(vmin, vmax)
``

Reply:
THanks for the suggestion @dopplershift. I just pushed that update.